### ISSUE TITLE
Test error that only happens on some (faster) computers

### ISSUE THREAD

**Issue by luis100** [keeps/roda#854]
The following test error happens consistently on my machine but I cannot reproduce on other machines which happen to be slower. Also, it does not happen on Travis build.

```
	
Tests run: 112, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 217.177 sec <<< FAILURE! - in TestSuite
testJobExecutingPluginThatStopsItselfUsingOrchestratorStop(org.roda.core.plugins.orchestrate.JobsTest)  Time elapsed: 2.097 sec  <<< FAILURE!
java.lang.AssertionError: 
Expected: is <STOPPED>
     but: was <COMPLETED>
	at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
	at org.roda.core.TestsHelper.executeJob(TestsHelper.java:112)
	at org.roda.core.TestsHelper.executeJob(TestsHelper.java:78)
	at org.roda.core.plugins.orchestrate.JobsTest.testJobExecutingPluginThatStopsItselfUsingOrchestratorStop(JobsTest.java:140)
```

**Comment by hsilva-keep** [comment]
Solved in c88d63fdcdf22d4684209a6286f83c29f44e2d99. @luis100 can you please test it in your computer?

**Comment by luis100** [comment]
It now works on my computer.
